bindings/pyroot/pythonizations/python/ROOT/_pythonization/_rdataframe.py

```diff
@@ -417,3 +417,175 @@ def pythonize_rdataframe(klass):
                                       HistoProfileWrapper,
                                       model_class)
         setattr(klass, method_name, getter)
+
+
+from . import pythonization
+from .._numbadeclare import _NumbaDeclareDecorator
+import re
+import warnings
+import numpy as np
+
+# Pythonizer 
+# def pythonize_define():
+    # @pythonization('RDataFrame', ns = 'ROOT')
+    # def pythonizer_rdf(klass):
+    #     klass._OriginalDefine =  klass.Define
+    #     klass.Define = _PyDefine
+
+
+@pythonization("RInterface<", ns="ROOT::RDF", is_prefix=True)
+def pythonize_rdataframeDefine(klass):
+    klass._OriginalDefine =  klass.Define
+    klass.PyDefine = _PyDefine
```

Comment:
The new implementation should replace the original `Define` (for which you keep a reference, and rightfully so). I.e. the user should not have to use `PyDefine` in their code, just `Define`.
The same applies for `Filter` below.